app/Domain/Clients/Templates/showClient.blade.php

```diff
@@ -194,9 +202,9 @@
 
                 <div id='comment'>
 
-                <form method="post" action="{{ BASE_URL }}/clients/showClient/<?php echo $tpl->e($_GET['id']); ?>#comment">
+                <form method="post" value="true" action="{{ BASE_URL }}/clients/showClient/<?php echo $tpl->e($values->id); ?>#comment">
```

Comment:
What does value="true" do in here? I don't think form has a "value" attribute

Reply:
without the value, the save property of array is set to null. In that case, the if condition in controller does not run